CRM/Price/Form/Preview.php

```diff
@@ -132,7 +132,9 @@ public function buildQuickForm() {
         foreach ($group['fields'] as $field) {
           $fieldId = $field['id'];
           $elementName = 'price_' . $fieldId;
-          CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          if (!empty($field['options'])) {
+            CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          }
```

Comment:
@jitendrapurohit i just re-tested on a locally re-built buildkit and couldn't reproduce the issue  as per attached image 
![price_text_field](https://cloud.githubusercontent.com/assets/6799125/24681857/4b1be982-19da-11e7-9613-07c6d71d4bfa.jpg)